src/cdk/activation.ts

```diff
@@ -22,9 +25,33 @@ export async function activate(activateArguments: { extensionContext: vscode.Ext
         treeDataProvider: explorer,
         showCollapseAll: true
     })
+
+    // Indicates workspace includes a CDK app and user has expanded the Node
+    view.onDidExpandElement(e => {
```

Comment:
This returns a disposable that should be passed to `activateArguments.extensionContext.subscriptions.push(...)` so that the event can be cleaned up.

Reply:
done